depclean-maven-plugin/src/main/java/se/kth/depclean/DepCleanMojo.java

```diff
@@ -194,6 +194,14 @@ private void printDependencies(Map<String, Long> sizeOfDependencies, Set<String>
         .forEachRemaining(s -> printString("\t" + s + " (" + getSize(s, sizeOfDependencies) + ")"));
   }
 
+  //Just a util function to print the information on the console
+  private void print_info_Dependencies(String info,Map<String,Long> sizeOfDependencies,Set<String> dependencies)
+  {
+    printString(info.toUpperCase()
+        + " [" + dependencies.size() + "]" + ": ");
+    printDependencies(sizeOfDependencies, dependencies);
+  }
```

Comment:
Should be like this:


```java
  /**
   * Add javadoc.
   * 
   * @param info Add javadoc.
   * @param sizeOfDependencies Add javadoc.
   * @param dependencies Add javadoc.
   */
  private void printInfoOfDependencies(String info, Map<String, Long> sizeOfDependencies, Set<String> dependencies) {
    printString(info.toUpperCase() + " [" + dependencies.size() + "]" + ": ");
    printDependencies(sizeOfDependencies, dependencies);
  }
```

Reply:
Thanks @cesarsotovalero @ABHAY0O7 for the acknowledgement. Surely I will get back with changes required.